src/microbe_stage/MicrobeStage.tscn

```diff
@@ -1543,9 +1550,6 @@ size_flags_horizontal = 0
 size_flags_vertical = 0
 texture = ExtResource( 57 )
 expand = true
-__meta__ = {
-"_edit_use_anchors_": false
-}
```

Comment:
Something was changed about the fonts (occurs several times), but I don't understand exactly what. I'll simply bring it to people more knowledgeable on front end to tell if this is a good thing or not.